src/main.js

```diff
@@ -236,7 +236,7 @@ const handlers = {
     updateServiceState();
     const statusLabel = status_labels[contents.code_status];
     document.getElementById("status").textContent = statusLabel;
-    document.title = `Robot - ${statusLabel}`;
+  document.title = `Robot - ${statusLabel || "Ready"}`;
```

Comment:
(Not sure why this whitespace was removed)
```suggestion
    document.title = `Robot - ${statusLabel || "Ready"}`;
```